docs/onboarding.rst

```diff
@@ -142,26 +131,32 @@ verify the journalist is set up for SecureDrop.
 
 The journalist should verify that they:
 
-1. Have their own *Journalist Tails USB* that they have verified they are able
```

Comment:
This section repeatedly italicizes terms that are not actually glossary terms. It may make sense to standardize the usage of *Admin Workstation USB* and *Journalist Workstation USB* and add it to the glossary; for now, I've just standardized on existing terminology.